docs/src/main/asciidoc/websockets-next-reference.adoc

```diff
@@ -846,6 +846,40 @@ quarkus.http.auth.permission.http-upgrade.paths=/end
 quarkus.http.auth.permission.http-upgrade.policy=authenticated
 ----
 
+Security annotations used during authentication must be placed on an endpoint class as well, for the `SecurityIdentity` is created before the websocket connection is opened.
+
+.Select Bearer token authentication mechanism
+[source, java]
+----
+package io.quarkus.websockets.next.test.security;
+
+import io.quarkus.oidc.BearerTokenAuthentication;
+import io.quarkus.websockets.next.OnTextMessage;
+import io.quarkus.websockets.next.WebSocket;
+
+@BearerTokenAuthentication <1>
+@WebSocket(path = "/end")
+public class Endpoint {
+
+    @OnTextMessage
+    String echo(String message) {
+        return message;
+    }
+
+}
+----
+<1> Require that opening WebSocket handshake request is authenticated using Bearer token authentication.
```

Comment:
```suggestion
<1> Require that an opening WebSocket handshake request is authenticated using the bearer token authentication.
```

As far as I recall, the Doc team prefers a lower case when referring to it in the text